README.md

```diff
@@ -0,0 +1,90 @@
+| `development`  | [![Build Status](https://travis-ci.org/NYPL/dgx-homepage.svg?branch=development)](https://travis-ci.org/NYPL/dgx-homepage)      |
+
+## Installation
+Install all npm dependencies listed under `package.json`
+```sh
+$ npm install
+```
+
+## Running the Application
+### Development Mode
+We use Webpack to fire off a hot-reloading development server. This allows for continuous code changes without the need to refresh your browser.
+
+```sh
+$ npm start // Starts localhost:3001 defaulting to APP_ENV=development
+```
+
+You can also set the `APP_ENV` variable which dictates what API environment to use as the main source.
+```sh
+$ APP_ENV=development|qa|production npm start // Starts localhost:3001 with set APP_ENV
+```
+
+### Production Mode
+To replicate the application in a production state. We execute 2 npm scripts with the proper `ENV` variables. By using `NODE_ENV=production`, we disable the hot-reload server. In addition, the `production` Webpack configuration by is set by running `npm run dist`.
+
+* **Step 1**: Build the distribution assets into `./dist/`
+```sh
+$ npm run dist
+```
+
+* **Step 2**: Starts the Node/Express server in `localhost:3001` with the `APP_ENV` set to `production`, targeting the proper `production` API's.
+```sh
+$ APP_ENV=production NODE_ENV=production npm start
+```
+
+## GIT Workflow
+We follow a [feature-branch](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) workflow. If you need to introduce/update the application code, you `SHOULD`:
+
+* Create a new branch off the `development` branch
+* Send a PR pointing to the `development` branch upon completion
+* Once the PR is approved, it should be merged into the `development` branch
+* If there are several PRs in process, a release should be scheduled by merging all completed PRs into the `development` branch
+* When a release is to be deployed, the `development` branch will be merged into `master`
+* All releases merged into `master` `MUST` be tagged and pushed to Github with their corresponding `version`
+
+## Components
+The homepage is made out of different React components that are displayed as features separated into rows. The React components are separate repos, so the homepage acts as a container that renders those components.
+
+Going from the top to the bottom of the page, we have the following list of components. All the components, **except** for the Content Banner component, are wrapped in the [dgx-homepage-row-component](git+ssh://git@bitbucket.org/NYPL/dgx-homepage-row-component.git) component. It is a container to render the feature for each row along with a left sidebar. Since the Content Banner takes up the whole width of the page, it is not wrapped in the dgx-homepage-row-component container.
+
+| Component | Repo |
+|---|---|
+| Content Banner | [dgx-homepage-content-banner](git+ssh://git@bitbucket.org/NYPL/dgx-homepage-content-banner) |
```

Comment:
Hmm can you remove the `git+ssh` in the urls here and just make it point to bitbucket? It doesn't work otherwise.